package.json

```
@@ -5,15 +5,17 @@
 	"author": "HPI Schul-Cloud",
 	"private": true,
 	"scripts": {
-		"dev:nuxt": "nodemon --inspect --watch src/ --ignore src/legacy-client src/server",
+		"dev:nuxt": "npx nodemon --inspect --watch src/ --ignore src/legacy-client src/server",
 		"dev:storybook": "start-storybook -c stories/.storybook -p 4001",
 		"dev:docs": "vuepress dev docs",
 		"dev": "concurrently \"yarn dev:nuxt\" \"yarn dev:storybook\" \"yarn dev:docs\" -n \"nuxt,story,docs\" -c \"green,magenta,cyan\" -i false --kill-others-on-fail",
 		"build:nuxt": "cross-env NODE_ENV=production nuxt build && cross-env sh ./copy_static.sh",
 		"build:docs": "vuepress build docs -d dist/docs",
 		"build:storybook": "build-storybook -c stories/.storybook -o dist/storybook",
 		"build": "run-s build:*",
+		"gulp": "cd src/legacy-client && npx gulp",
```

Maybe:
```suggestion
		"build:nuxt:gulp": "cd src/legacy-client && npx gulp",
```
to make clear, that this build process is used for nuxt and not storybook or vuepress?